docs/reference/ingress/kubernetes-with-cert-manager.md

```diff
@@ -0,0 +1,168 @@
+# Deploying with SSL on Kubernetes using Cert-Manager and Nginx
```

Comment:
I think the reference title should be something like `ssl-kubernetes` since cert-manager is too specific. For instance we changed from Caddy to this recently and we'd have to break the permalinks doing that.

Reply:
This was meant to be very specific to cert manager. In our conversation the other day we talked about having several sets of instructions depending on different cluster configurations (e.g. Caddy or Traefik).  If you are suggesting that we would promote cert-manager as _the_ way to do this with no other instructions, then I am ok with renaming it.